src/components/CurrentUserPersonalDetailsSkeletonView/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SkeletonViewContentLoader from 'react-content-loader/native';
+import {Circle, Rect} from 'react-native-svg';
+import {View} from 'react-native';
+import * as StyleUtils from '../../styles/StyleUtils';
+import CONST from '../../CONST';
+import themeColors from '../../styles/themes/default';
+import variables from '../../styles/variables';
+import styles from '../../styles/styles';
+
+const propTypes = {
+    /** Whether to animate the skeleton view */
+    shouldAnimate: PropTypes.bool,
+};
+
+const defaultProps = {
+    shouldAnimate: true,
+};
+
+function CurrentUserPersonalDetailsSkeletonView(props) {
+    const avatarPlaceholderSize = StyleUtils.getAvatarSize(CONST.AVATAR_SIZE.LARGE);
+    const avatarPlaceholderRadius = avatarPlaceholderSize / 2;
+    const headlineMarginTop = 16;
+    const headlineSize = variables.fontSizeXLarge;
```

Comment:
I think there's enough of a height difference that we should address it.

The skeleton has a height of 155:
![Screenshot 2023-07-10 at 2 29 30 PM](https://github.com/Expensify/App/assets/7277067/8e76ae4d-b272-4d09-97ca-565741eacd69)

And once loaded it's 162:
![Screenshot 2023-07-10 at 2 28 39 PM](https://github.com/Expensify/App/assets/7277067/31fd0c92-493f-4c68-8b71-d99163f6433e)

Furthermore, `const headlineMarginTop = 16;` and `const labelMarginTop = 4;` are kind of "magic" numbers at the moment and could get out of sync if we were to change the styles of the real components.

I think it'd be better to replace these 2 with something more precise, e.g. instead of `headlineMarginTop`, do something like:
```
const spaceBetweenAvatarAndHeadline = styles.mb3.marginBottom + styles.mt1.marginTop + ((variables.lineHeightXXLarge - variables.fontSizeXLarge)/ 2);
```
and then similarly for `labelMarginTop`, with some comments explaining the calculations.

Thoughts?